modules/home/my_home/default.nix

```diff
@@ -24,114 +23,111 @@ with lib;
       };
     };
   };
-    config = mkIf (cfg.username != "") {
-      home-manager = {
-        #TODO: sort out whether we want global or user packages
-        # useUserPackages = true;
-        useGlobalPkgs = true;
```

Comment:
There might be a misunderstanding here. You _probably_ want `useGlobalPkgs` since it uses the same package set from NixOS/nix-darwin. The `useUserPackages` option makes it so adding packages to `users.users.<name>.packages` installs them with Home Manager. You probably want that as well.